### PR TITLE
Issue 2169 - BackupFileDisclosure Unit Tests

### PR DESCRIPTION
**Pull request by kingthorin**
BackupFileDisclosure > Add unit tests for message counts at different strengths.
ActiveScannerTest > Aligned to the latest changes in the master branch in order to handle scanner configuration, etc.

See zaproxy/zaproxy#2169

### CODE REVIEW

File: test/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureUnitTest.java [hunk 1]
```diff
@@ -43,10 +46,73 @@
             + "<p>You don't have permission to access " + PATH_TOKEN + "\n"
             + "on this server.</p>\n"
             + "</body></html>";
+    private static final String URL = "/index.html";
```
Comment (thc202) [inline test/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureUnitTest.java]: Having an extra "directory" (e.g. `/dir/index.html`) increases the coverage and adds some more requests (still under the limits).

**Comment by kingthorin** [comment]
Ok I've got that addressed. Interestingly I accidentally ran all the tests in the master branch and the XSS tests failed big time, are they running ok for you locally?

**Comment by kingthorin** [comment]
Shouldn't this path:
https://github.com/zaproxy/zap-extensions/blob/dafb0dc21b0a66de0d59fe205b8380d95f44f878/test/org/zaproxy/zap/extension/ascanrules/ActiveScannerTest.java#L91

Be:
`private static final String BASE_RESOURCE_DIR = "/resources/org/zaproxy/zap/extension/ascanrules/";`

That seems to make things happy locally.

**Comment by thc202** [comment]
> are they running ok for you locally?

They are failing when running from Eclipse, classpath needs to be changed (it is running fine with Ant, Gradle, and Eclipse+Gradle though). I also tested in Windows and noticed that the path is not correct, I'll fix both issues shortly.

**Comment by thc202** [comment]
Done in #1573.

**Comment by thc202** [comment]
Would you mind updating the test class with the changes from the other PR? Note that you can also delete the files in "test/resources/install", no longer needed.

**Comment by kingthorin** [comment]
Will do.

**Comment by thc202** [comment]
Thanks.

**Comment by thc202** [comment]
Thank you!

**Pull request merged**